Services/CO.CDP.OrganisationInformation.Persistence/OrganisationInformationContext.cs

```diff
@@ -2,15 +2,23 @@
 
 namespace CO.CDP.OrganisationInformation.Persistence;
 
-public class OrganisationInformationContext(string connectionString) : DbContext
+public class OrganisationInformationContext : DbContext
 {
-    public DbSet<Tenant> Tenants { get; set; } = null!;
-    public DbSet<Organisation> Organisations { get; set; } = null!;
-    public DbSet<Person> Persons { get; set; } = null!;
+    private readonly string _connectionString;
+
+    public OrganisationInformationContext(string connectionString)
+    {
+        _connectionString = connectionString;
+    }
```

Comment:
Seems more verbose. Any reason for this change?

Reply:
Ignore this, was part of debugging the issues with EF context